asistente_ladm_col/core/supplies/etl_snc.py

```diff
@@ -138,6 +145,7 @@ def run_etl_model(self, custom_feedback):
                         'manzana': self.gdb_layer_paths['U_MANZANA'],
                         'persona': self.alphanumeric_file_paths['persona'],
                         'personapredio': self.alphanumeric_file_paths['persona_predio'],
+                        'personapredio2': self.alphanumeric_file_paths['predio_bloqueo'],
```

Comment:
¿`personapredio2`?

Reply:
This is curious because I chose the name PREDIO_BLOQUEO, but QGIS choose personapredio2, I don know why?

Look it 

![image](https://user-images.githubusercontent.com/27906888/86796202-f71d9e80-c033-11ea-95c2-ed869e9b32ed.png)